tests/ui/test_other_user.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework.testlib import ManageTest, ui
+from time import sleep
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class OBCUI(PageNavigator):
+    def __init__(self, driver):
+        super().__init__(driver)
+        ocp_version = get_ocp_version()
+        self.sc_loc = locators[ocp_version]["OBC"]
+
+    def check_oc_option(self):
+        self.navigate_storageclasses_page()
+
+        "check OBC is visible to user after giving admin access"
+
+        self.do_click(self.sc_loc["create_obc"])
+
+
+class TestOBCUi(ManageTest):
+    @ui
+    def test_create_storageclass_rbd(self, user_factory, login_factory):
+
+        "Create user"
+        user = user_factory()
+        kubeconfig = os.getenv("KUBECONFIG")
+        kube_data = ""
+        with open(kubeconfig, "r") as kube_file:
+            kube_data = kube_file.readlines()
+        sleep(30)
+        exec_cmd(["oc", "login", "-u", user[0], "-p", user[1]], secrets=[user[1]])
+        exec_cmd(["oc", "logout"])
+        with open(kubeconfig, "w") as kube_file:
+            kube_file.writelines(kube_data)
+
+        "Login using created user"
+        login_factory(user[0], user[1])
+        obc_ui_obj = OBCUI()
+        obc_ui_obj.Check_OBC_option()
```

Comment:
The function you created in the class is called `check_oc_option` so you are calling a None existing class.